lib/polyglot.rb

```diff
@@ -93,6 +93,19 @@ def process_active_language
         langs[url] = language
       end
       site.posts.docs = approved.values
+      approved = {}
```

Comment:
This code is basically the same as above. It's not [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).